src/bin/hq.rs

```diff
@@ -238,6 +239,7 @@ async fn command_job_detail(gsettings: GlobalSettings, opts: JobDetailOpts) -> a
                 }
             }
         }
+        JobSpecifier::All => return Err(Error::msg("Specifier all is not implemented for job details, did you mean: job list?"))
```

Comment:
It's enough to use `bail!("Error message")` or `return Err(anyhow!("Error message"))`.